lib/Environment.js

```diff
@@ -3,6 +3,8 @@ var vow = require('vow'),
 
 function Environment(options) {
     debug('Initializing...');
+
+    this._errbacks = [];
```

Comment:
errback is callback for error )) In this particular case it's errbacks for `evaluateJs` fucntions. Sometimes we need to know that something going wrong. For example: if we use `wait` and on first iteration code fails, we should stop waiting. We add errback before `wait` and remove it when `wait` is completed.